april heartbeat

jendefig opened the pull request:
April Heartbeat Draft!

jorgeorpinel commented:
Missing a cover img?
![image](https://user-images.githubusercontent.com/1477535/114788507-32799600-9d47-11eb-8cb2-85a2014bd7c8.png)

jorgeorpinel commented:
BTW lol
![image](https://user-images.githubusercontent.com/1477535/114788586-576e0900-9d47-11eb-939b-b681d3c4d7e2.png)

jorgeorpinel commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,182 @@
+
+First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of
+[Made With ML](https://madewithml.com/) has created
+[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.  
+It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```
Was that new line intentional?
```suggestion
[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.
It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```

jorgeorpinel replied:
p.s. breaks the p:
![image](https://user-images.githubusercontent.com/1477535/114788876-dbc08c00-9d47-11eb-8a2c-ab6667d5e97b.png)

jendefig commented:
> Missing a cover img?
> ![image](https://user-images.githubusercontent.com/1477535/114788507-32799600-9d47-11eb-8cb2-85a2014bd7c8.png)

Got it.  I had it mis-named.

jorgeorpinel reviewed (approved):
LGTM besides the couple details noted.

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,182 @@
+<external-link
+href="https://madewithml.com/courses/mlops/"
+title="MLOps"
+description="DVC tutorial"
```
could we put some better descriptions in these items please?

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,182 @@
+likes to spend time with his family away from screens (😅🙌🏼) and he volunteers
+for the state emergency service. 🤲🏼
+
+[**Yanxiang Gao**](https://www.linkedin.com/in/gaoyanxiang/)(who graciously
```
💅 need space before `(who ...`

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,182 @@
+
+![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)
+
+### Goku Mohandas Tutorial
```
I think he calls them lessons (it's more of a course) - we can use the same terminology. May be mentioned that it is part of the course

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,182 @@
+  with DVC, our CEO's interviews and talks, new team members and more!
+picture: 2021-04-15/apr21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718?u=jendefig
```
`?u=jendefig` - no need for this

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,182 @@
+title: April '21 Heartbeat
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 5 new hires and more!
```
💅 please try to keep under 80 symbols

shcheklein reviewed (changes requested):
Great blog post!

- A few minor comments here and there
- We have some image duplicates it seems
- Some images look too large for the task the solve, let's resize them accordingly

jendefig commented:
@shcheklein I think I've covered everything.  I changed the image size of the Spanish article.  I attempted to downsize the first gif, but had to house it else where and the resulting link did not work, so I put it back. The header image should be corrected.  I added tags and made all the other changes.  I think the duplicate images issue should be fixed.  

I don't know why it's failing.  I had a .json file that appeared from the beginning and I was ignoring, but it was part of one of the recent commits when I was attempting to figure out how do do this in VSCode.  Let me know what I need to fix.

jorgeorpinel commented:
@jendefig please delete /package-lock.json . Remember to aviod `npm` 🙂

jorgeorpinel commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,187 @@
+date: 2021-04-15
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires and more!
```
💅 
```suggestion
  5 new hires, and more!
```

jendefig replied:
You guys are really comma happy. Ok.  I will submit.

jendefig replied:
Also added comment after member in long version for consistency.  😉

jorgeorpinel replied:
Haha. I usually avoid going into stylistic suggestions but I thought "5 new hired and more" sounded a bit like "more than 5 new hires".

jorgeorpinel commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,187 @@
+description: |
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires and more!
+descriptionLong: |
+  Monthly updates are here! Read all about what our community is doing
+  with DVC, our CEO's interviews and talks, new team members and more!
```
Didn't expand on use cases or videos here.

jorgeorpinel replied:
BTW I'm not sure where short vs long descriptions display. Is the short one for SEO? If so it should have a max length, probably? (which I think we're within here, just curious what are the guidelines here)

jendefig replied:
I think the short description is on the main blog page post card and the long description is at the top when you click into it.  Not sure which works with SEO 🤷🏻‍♀️ @rogermparent, can you help us with SEO and how we might be thinking about this in this respect?

jendefig replied:
Let me know if you think this still needs further extending @jorgeorpinel !

rogermparent replied:
> the short description is on the main blog page post card and the long description is at the top when you click into it.

Yep! Also, all the SEO meta tags use the short description. As far as the content, the only big thing to consider when making a description from this standpoint is to prioritize making a good description that draws attention over stuffing it with keywords (though some keywords in the right place can draw attention!), as Google doesn't consider meta description keywords in its ranking algorithm.

![seo-screenshot](https://user-images.githubusercontent.com/9111807/115066881-206e3380-9ebe-11eb-9044-4a795d9f8a80.png)

jorgeorpinel replied:
> the short description is on the main blog page post card and the long description is at the top when you click into it

Ah yes now that the PR is deploying I can confirm that. Thanks for the reminder! 

> Not sure which works with SEO 🤷🏻‍♀️ 

Yep. It looks like the short one will go into social media previous (good to know). As for Google search results, they do whatever they want anyway — sometimes they use the suggested title/description, sometimes they build their own.

BTW (since we're tagging @rogermparent 😬) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:

![image](https://user-images.githubusercontent.com/1477535/115067099-0d576580-9eb6-11eb-85ff-ca2bee556a43.png)

jorgeorpinel replied:
p.s. just noticed your answer after posting mine, @rogermparent.

jorgeorpinel replied:
> making a good description that draws attention over stuffing it with keywords

As long as they match the content yes, otherwise the algo may even penalize ranking for trying to mislead it 😋 or so I've heard.

jendefig replied:
> BTW (since we're tagging @rogermparent 😬) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:

I agree, there's definitely space for it, I can change.  @rogermparent, on the short description for SEO purposes are there length guidelines?  What = short?

jorgeorpinel replied:
> What = short?

Probably whatever fits comfortably in our blog cards will roughly work for social media previews too:

![image](https://user-images.githubusercontent.com/1477535/115068620-1a755400-9eb8-11eb-9d52-aa59b55dc7fd.png)

rogermparent replied:
> BTW (since we're tagging @rogermparent grimacing) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:
> 
> ![image](https://user-images.githubusercontent.com/1477535/115067099-0d576580-9eb6-11eb-85ff-ca2bee556a43.png)

That's a good idea, we should be able to fetch the long description of the first article in the query and show it on the first card of these index pages. If nothing else, we can make a ticket for the future.

jorgeorpinel replied:
https://github.com/iterative/dvc.org/issues/2386

jorgeorpinel commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,187 @@
+### Goku Mohandas' New Lessons!
+
+First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of
+[Made With ML](https://madewithml.com/) has created
+[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.  
+It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```
Per https://github.com/iterative/dvc.org/pull/2372#discussion_r613693094:

```suggestion
[Made With ML](https://madewithml.com/) has added this
[Versioning](https://madewithml.com/courses/mlops/versioning/) lesson to the
MLOps course using DVC. It's RT'ing around the ML Twitter space like hotcakes!
🥞
```

jorgeorpinel replied:
p.s. please retweet some hotcakes my way.

jendefig replied:
Much better.  Thank you!  I'm correcting this way on my side and not committing suggestion because that's how I usually get in to trouble.  Trying to get this out the door..... Maybe that makes this more complicated?

jorgeorpinel reviewed (commented):
Just a couple things based on @shcheklein's review. Thanks 👇

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,190 @@
+<external-link
+href="https://makerspace.aisingapore.org/2021/04/data-versioning-for-cd4ml-part-2/"
+title="Data Versioning for CD4ML - Part 2"
+description="DVC tutorial"
```
we still have the same `DVC tutorial` description in all these snippets (except the first one). Could we please make them a bit longer and specific to the item?

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,190 @@
+  Monthly updates are here! Loads from the Community, use cases, videos, 
+  5 new hires, and more!
+descriptionLong: |
+  Monthly updates are here! Read all about new lessons and a tutorials from
```
btw, just a tip - this section supports links, and _different_ **text** `styles`

shcheklein reviewed (changes requested):
Looks great! Just one last thing with the link descriptions

shcheklein commented on content/blog/2021-04-15-april-21-dvc-heartbeat.md:
```diff
@@ -0,0 +1,192 @@
+﻿---
+title: April '21 Heartbeat
```
Merged, but I see that we should also change the file name ^^

and images like `/uploads/images/2021-04-15/made-with-ml-logo.png` - I'm not even sure why it does work now
